.travis.yml

```diff
@@ -6,12 +9,7 @@ rvm:
 env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
 
 before_script:
-  - psql -c 'create database timeoverflow_test;' -U postgres
-  - RAILS_ENV=test bundle exec rake db:migrate
-
-sudo: false
-
-cache: bundler
+  - bundle exec rake db:setup db:migrate
```

Comment:
yeah you're absolutely right! :+1:
I'll amend it  tomorrow
On 22 Mar 2016 9:10 pm, "Enrico Stano" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/coopdevs/timeoverflow/pull/191#discussion_r57059750>:
>
> > @@ -6,12 +9,7 @@ rvm:
> >  env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
> >
> >  before_script:
> > -  - psql -c 'create database timeoverflow_test;' -U postgres
> > -  - RAILS_ENV=test bundle exec rake db:migrate
> > -
> > -sudo: false
> > -
> > -cache: bundler
> > +  - bundle exec rake db:setup db:migrate
>
> Every time we add a new migration, in the same PR we add the modified
> schema. We must ensure this. We generally enforce this running a bad
> practices suite. I would rely on our capability to don't break the
> schema, even more.. if we break the schema is better to know that in travis
> testing a PR than directly in production.
>
> What do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coopdevs/timeoverflow/pull/191/files/8249b2b56da19f4e66107eb4fee4e13e34c2c360#r57059750>
>